28.10.16 08:38:51.632 [main] ERROR o.s.t.c.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@a2f7f9] to prepare test instance [jd.toll.server.dao.DaoTest@eb2c50]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:122)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:105)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:74)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:312)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:211)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:288)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:284)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:231)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:88)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:174)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in URL [file:/C:/Projects/tollgit/toll/server/build/resources/main/applicationContext.xml]: Invocation of init method failed; nested exception is liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.monitor-new-nodes.xml::db.monitor-new-nodes::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1512)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:521)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:458)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:223)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:293)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:628)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:479)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:120)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 29 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.monitor-new-nodes.xml::db.monitor-new-nodes::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:376)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:40)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:64)
	at liquibase.Liquibase.update(Liquibase.java:137)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:324)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:291)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1571)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1509)
	... 42 common frames omitted
Caused by: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:103)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1513)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1496)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:343)
	... 49 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLException: Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.index.BaseIndex.getDuplicateKeyException(BaseIndex.java:102)
	at org.h2.index.PageBtree.find(PageBtree.java:120)
	at org.h2.index.PageBtreeLeaf.addRow(PageBtreeLeaf.java:150)
	at org.h2.index.PageBtreeLeaf.addRowTry(PageBtreeLeaf.java:103)
	at org.h2.index.PageBtreeIndex.addRow(PageBtreeIndex.java:96)
	at org.h2.index.PageBtreeIndex.add(PageBtreeIndex.java:87)
	at org.h2.table.RegularTable.addRow(RegularTable.java:119)
	at org.h2.command.dml.Insert.insertRows(Insert.java:156)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at liquibase.executor.jvm.JdbcExecutor$1ExecuteStatementCallback.doInStatement(JdbcExecutor.java:90)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:50)
	... 53 common frames omitted
28.10.16 08:39:21.909 [main] ERROR o.s.t.c.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@a2f7f9] to prepare test instance [jd.toll.server.dao.DaoTest@eb2c50]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:122)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:105)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:74)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:312)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:211)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:288)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:284)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:231)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:88)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:174)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in URL [file:/C:/Projects/tollgit/toll/server/build/resources/main/applicationContext.xml]: Invocation of init method failed; nested exception is liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.monitor-new-nodes.xml::db.monitor-new-nodes::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1512)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:521)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:458)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:223)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:293)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:628)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:479)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:120)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 29 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.monitor-new-nodes.xml::db.monitor-new-nodes::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:376)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:40)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:64)
	at liquibase.Liquibase.update(Liquibase.java:137)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:324)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:291)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1571)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1509)
	... 42 common frames omitted
Caused by: liquibase.exception.DatabaseException: Error executing SQL insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:103)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1513)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1496)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:343)
	... 49 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLException: Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"
Unique index or primary key violation: "PRIMARY_KEY_4 ON PUBLIC.NODES(ZIG_BEE_ID_64) VALUES ( /* 11 */ '0013A20040F9B0F5' )"; SQL statement:
insert into nodes (ZIG_BEE_ID_64, CREATE_TIME_STAMP, UPDATE_TIME_STAMP, NODE_NAME) values ('0013A20040F9B0F5', 1466431769329, 1466431769329, 'B0F5') [23505-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.index.BaseIndex.getDuplicateKeyException(BaseIndex.java:102)
	at org.h2.index.PageBtree.find(PageBtree.java:120)
	at org.h2.index.PageBtreeLeaf.addRow(PageBtreeLeaf.java:150)
	at org.h2.index.PageBtreeLeaf.addRowTry(PageBtreeLeaf.java:103)
	at org.h2.index.PageBtreeIndex.addRow(PageBtreeIndex.java:96)
	at org.h2.index.PageBtreeIndex.add(PageBtreeIndex.java:87)
	at org.h2.table.RegularTable.addRow(RegularTable.java:119)
	at org.h2.command.dml.Insert.insertRows(Insert.java:156)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at liquibase.executor.jvm.JdbcExecutor$1ExecuteStatementCallback.doInStatement(JdbcExecutor.java:90)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:50)
	... 53 common frames omitted
28.10.16 09:16:23.712 [main] WARN  o.h.e.j.s.SqlExceptionHelper - SQL Error: 42102, SQLState: 42S02
28.10.16 09:16:23.728 [main] ERROR o.h.e.j.s.SqlExceptionHelper - Таблица "OWNER" не найдена
Table "OWNER" not found; SQL statement:
select xbeeowner0_.id as id1_1_, xbeeowner0_.drivelisence as drivelis2_1_, xbeeowner0_.firstname as firstnam3_1_, xbeeowner0_.lastname as lastname4_1_, xbeeowner0_.middlename as middlena5_1_, xbeeowner0_.passport as passport6_1_ from owner xbeeowner0_ [42102-185]
28.10.16 09:18:44.254 [main] WARN  o.h.e.j.s.SqlExceptionHelper - SQL Error: 42102, SQLState: 42S02
28.10.16 09:18:44.270 [main] ERROR o.h.e.j.s.SqlExceptionHelper - Таблица "OWNER" не найдена
Table "OWNER" not found; SQL statement:
select xbeeowner0_.id as id1_1_, xbeeowner0_.drivelisence as drivelis2_1_, xbeeowner0_.firstname as firstnam3_1_, xbeeowner0_.lastname as lastname4_1_, xbeeowner0_.middlename as middlena5_1_, xbeeowner0_.passport as passport6_1_ from owner xbeeowner0_ [42102-185]
28.10.16 09:31:29.379 [main] WARN  o.h.e.j.s.SqlExceptionHelper - SQL Error: 42122, SQLState: 42S22
28.10.16 09:31:29.379 [main] ERROR o.h.e.j.s.SqlExceptionHelper - Столбец "XBEEOWNER0_.DRIVELISENCE" не найден
Column "XBEEOWNER0_.DRIVELISENCE" not found; SQL statement:
select xbeeowner0_.id as id1_1_, xbeeowner0_.drivelisence as drivelis2_1_, xbeeowner0_.firstname as firstnam3_1_, xbeeowner0_.lastname as lastname4_1_, xbeeowner0_.middlename as middlena5_1_, xbeeowner0_.passport as passport6_1_ from owner xbeeowner0_ [42122-185]
28.10.16 09:31:52.077 [main] WARN  o.h.e.j.s.SqlExceptionHelper - SQL Error: 42122, SQLState: 42S22
28.10.16 09:31:52.077 [main] ERROR o.h.e.j.s.SqlExceptionHelper - Столбец "XBEEOWNER0_.DRIVELISENCE" не найден
Column "XBEEOWNER0_.DRIVELISENCE" not found; SQL statement:
select xbeeowner0_.id as id1_1_, xbeeowner0_.drivelisence as drivelis2_1_, xbeeowner0_.firstname as firstnam3_1_, xbeeowner0_.lastname as lastname4_1_, xbeeowner0_.middlename as middlena5_1_, xbeeowner0_.passport as passport6_1_ from owner xbeeowner0_ [42122-185]
28.10.16 09:56:59.559 [main] ERROR o.s.t.c.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@15e710b] to prepare test instance [jd.toll.server.dao.DaoTest@f7cdf8]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:122)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:105)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:74)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:312)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:211)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:288)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:284)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:231)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:88)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:174)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in URL [file:/C:/Projects/tollgit/toll/server/build/resources/main/applicationContext.xml]: Invocation of init method failed; nested exception is liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-transport.xml::db.server-new-transport::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1'): Нарушение ссылочной целостности: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"
Referential integrity constraint violation: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"; SQL statement:
insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1') [23506-185]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1512)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:521)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:458)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:223)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:293)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:628)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:479)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:120)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 29 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-transport.xml::db.server-new-transport::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1'): Нарушение ссылочной целостности: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"
Referential integrity constraint violation: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"; SQL statement:
insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1') [23506-185]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:376)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:40)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:64)
	at liquibase.Liquibase.update(Liquibase.java:137)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:324)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:291)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1571)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1509)
	... 42 common frames omitted
Caused by: liquibase.exception.DatabaseException: Error executing SQL insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1'): Нарушение ссылочной целостности: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"
Referential integrity constraint violation: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"; SQL statement:
insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1') [23506-185]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:103)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1513)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1496)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:343)
	... 49 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLException: Нарушение ссылочной целостности: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"
Referential integrity constraint violation: "TRANSPORT_OWNER_ID_FK: PUBLIC.TRANSPORT FOREIGN KEY(OWNER_ID) REFERENCES PUBLIC.OWNER(ID) ('0013A20040F9C1D1')"; SQL statement:
insert into transport (TYPE, MODEL, BRAND, STATENUMBER, ID, OWNER_ID) values ('auto', '2106', 'lada', 'b105dv70', '0013A20040F9B0F8' , '0013A20040F9C1D1') [23506-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.constraint.ConstraintReferential.checkRowOwnTable(ConstraintReferential.java:372)
	at org.h2.constraint.ConstraintReferential.checkRow(ConstraintReferential.java:314)
	at org.h2.table.Table.fireConstraints(Table.java:920)
	at org.h2.table.Table.fireAfterRow(Table.java:938)
	at org.h2.command.dml.Insert.insertRows(Insert.java:161)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at liquibase.executor.jvm.JdbcExecutor$1ExecuteStatementCallback.doInStatement(JdbcExecutor.java:90)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:50)
	... 53 common frames omitted
28.10.16 10:04:25.453 [main] ERROR o.s.t.c.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@1202c31] to prepare test instance [jd.toll.server.dao.DaoTest@182d777]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:122)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:105)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:74)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:312)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:211)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:288)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:284)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:231)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:88)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:174)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in URL [file:/C:/Projects/tollgit/toll/server/build/resources/main/applicationContext.xml]: Invocation of init method failed; nested exception is liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-owner.xml::db.server-new-owner::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1'): Значение слишком длинное для поля "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"
Value too long for column "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1') [22001-185]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1512)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:521)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:458)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:223)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:293)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:628)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:479)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:120)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 29 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-owner.xml::db.server-new-owner::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1'): Значение слишком длинное для поля "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"
Value too long for column "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1') [22001-185]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:376)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:40)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:64)
	at liquibase.Liquibase.update(Liquibase.java:137)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:324)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:291)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1571)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1509)
	... 42 common frames omitted
Caused by: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1'): Значение слишком длинное для поля "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"
Value too long for column "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1') [22001-185]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:103)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1513)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1496)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:343)
	... 49 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLException: Значение слишком длинное для поля "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"
Value too long for column "DRIVELICENSE VARCHAR(10)": "'hdfh45454fd' (11)"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh45454fd', '0013A20040F9C1D1') [22001-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:327)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:737)
	at org.h2.command.dml.Insert.insertRows(Insert.java:151)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at liquibase.executor.jvm.JdbcExecutor$1ExecuteStatementCallback.doInStatement(JdbcExecutor.java:90)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:50)
	... 53 common frames omitted
28.10.16 10:05:13.025 [main] ERROR o.s.t.c.TestContextManager - Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@1824872] to prepare test instance [jd.toll.server.dao.DaoTest@1016b4e]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.TestContext.getApplicationContext(TestContext.java:122)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:105)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:74)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:312)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:211)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:288)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:284)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:231)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:88)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:71)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:174)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'liquibase' defined in URL [file:/C:/Projects/tollgit/toll/server/build/resources/main/applicationContext.xml]: Invocation of init method failed; nested exception is liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-owner.xml::db.server-new-owner::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"
Unique index or primary key violation: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1') [23505-185]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1512)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:521)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:458)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:296)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:223)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:293)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:628)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:932)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:479)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:120)
	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContextInternal(CacheAwareContextLoaderDelegate.java:64)
	at org.springframework.test.context.CacheAwareContextLoaderDelegate.loadContext(CacheAwareContextLoaderDelegate.java:91)
	... 29 common frames omitted
Caused by: liquibase.exception.MigrationFailedException: Migration failed for change set /dblog/db.server-new-owner.xml::db.server-new-owner::saturn:
     Reason: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"
Unique index or primary key violation: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1') [23505-185]
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:376)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:40)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:64)
	at liquibase.Liquibase.update(Liquibase.java:137)
	at liquibase.integration.spring.SpringLiquibase.performUpdate(SpringLiquibase.java:324)
	at liquibase.integration.spring.SpringLiquibase.afterPropertiesSet(SpringLiquibase.java:291)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1571)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1509)
	... 42 common frames omitted
Caused by: liquibase.exception.DatabaseException: Error executing SQL insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1'): Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"
Unique index or primary key violation: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1') [23505-185]
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:57)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:103)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1513)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1496)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:343)
	... 49 common frames omitted
Caused by: org.h2.jdbc.JdbcSQLException: Нарушение уникального индекса или первичного ключа: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"
Unique index or primary key violation: "PRIMARY_KEY_48 ON PUBLIC.OWNER(ID) VALUES ( /* 3 */ '0013A20040F9C2D1' )"; SQL statement:
insert into owner (LASTNAME, FIRSTNAME, MIDDLENAME, PASSPORT, DRIVELICENSE, ID) values ('Ivanov', 'Ivan', 'Ivanovich', '9696112233', 'hdfh4545fd', '0013A20040F9C2D1') [23505-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.index.BaseIndex.getDuplicateKeyException(BaseIndex.java:102)
	at org.h2.index.PageBtree.find(PageBtree.java:120)
	at org.h2.index.PageBtreeLeaf.addRow(PageBtreeLeaf.java:150)
	at org.h2.index.PageBtreeLeaf.addRowTry(PageBtreeLeaf.java:103)
	at org.h2.index.PageBtreeIndex.addRow(PageBtreeIndex.java:96)
	at org.h2.index.PageBtreeIndex.add(PageBtreeIndex.java:87)
	at org.h2.table.RegularTable.addRow(RegularTable.java:119)
	at org.h2.command.dml.Insert.insertRows(Insert.java:156)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at liquibase.executor.jvm.JdbcExecutor$1ExecuteStatementCallback.doInStatement(JdbcExecutor.java:90)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:50)
	... 53 common frames omitted
